src/bundle/Resources/views/content/tab/url/modal_add_customurl.html.twig

```diff
@@ -0,0 +1,42 @@
+{% trans_default_domain 'locationview' %}
```

Comment:
Please use dedicated domain for translations. No need to push everything into `locationview`, it only makes this domain bloated with too much stuff.